Makefile

```diff
@@ -12,6 +12,8 @@ MONOPATH=/Library/Frameworks/Mono.framework/Libraries/mono/4.5/
 NUNITCONSOLE="/Library/Frameworks/Mono.framework/Versions/Current/bin/nunit-console4"
 MONOBIN=/Library/Frameworks/Mono.framework/Commands
 NUGET=$(MONOBIN)/nuget
+PIP=pip
+PYTHON=python
```

Comment:
I would like to keep Python 3 as the default.
However it is possible to set the variables through `make` conditionals. For instance:

    pyversion ?= 3
    ifeq (3,${pyversion})
    PYTHON = python3
    ... etc ...
    else
    PYTHON = python
    ... etc ...
    endif

Then you can change the default build by doing `make pyversion=2 pydemo`, instead of just `make pydemo`.